backend/testing/configs/config2.yaml

```diff
@@ -5,6 +5,10 @@ monitors:
     riperis: ['rrc01']
     bgpstreamlive: ['routeviews', 'ris']
     betabmp: ['betabmp']
+    bgpstreamhist: ./
+    exabgp:
+        - ip: exabgp
```

Comment:
just check of this setting generates any errors since no exabgp container is used/configured during testing (I have not tried to see what happens when you activate the monitor with incorrect configuration, maybe it is ok for the rest of the system).